pact/message_provider.py

```diff
@@ -115,6 +115,25 @@ def verify(self):
         return_code, _ = verifier.verify_pacts(pact_files, verbose=False)
         assert (return_code == 0), f'Expected returned_code = 0, actual = {return_code}'
 
+    def verify_with_broker(self, enable_pending=False, include_wip_pacts_since=None, **kwargs):
```

Comment:
Is this user facing code? It's missing the consumer version selectors. How can we get those supported?

Reply:
The API docs for fetching the pacts for verification are here: https://github.com/pact-foundation/pact_broker/blob/master/lib/pact_broker/doc/views/provider-pacts-for-verification.markdown